README.md

```diff
@@ -0,0 +1,14 @@
+# Moby
+
+[Moby Project](https://mobyproject.org)
+
+The Moby Project is an open framework to assemble specialized container systems without reinventing the wheel.
+
+Moby is an open framework created by Docker to assemble specialized container systems without reinventing the wheel. It provides a “lego set” of dozens of standard components and a framework for assembling them into custom platforms. At the core of Moby is a framework to assemble specialized container systems which provides:
```

Comment:
provides what?

Reply:
Ugh, that copy got messed up. I had copied the basics off of mobyproject.org.